Nos/Views/ProfileHeader.swift

```diff
@@ -74,6 +76,11 @@ struct ProfileHeader: View {
                         } label: {
                             Text("\(Localized.following.string): \(author.follows?.count ?? 0)")
                         }
+                        
+                        Spacer()
+                        Text("\(Localized.nip05.string): \(verifiedNip05Identifier)")
```

Comment:
The verifiedNip05Identifier is initialized as an empty string, and then is updated in .onAppear after it has been verified.